frontend/src/features/books/BookDetails.jsx

```diff
@@ -30,13 +25,33 @@ function BookDetails() {
     fetchCurrentBook()
   }, [id])
 
-  if (!title) return <h3>Loading...</h3>
+  const deleteBook = async () => {
+    try {
+      const response = await fetch(`${API_URL}/${id}`, {
+        method: "DELETE"
+      })
+      if (response.ok) {
+        navigate('/')
+      } else {
+        const errorText = await response.text()
+        throw new Error(`Error ${response.status}: ${errorText}`)
+      }
+    } catch (e) {
+      console.log("An error occurred", e)
```

Comment:
@nshelokar-vf, rather than logging error, can we use toast to show error message?

Reply:
Yes we can. I will add that.